src/operators/skip-property-null.ts

```diff
@@ -14,14 +14,20 @@ import { filter } from "rxjs/operators";
  *
  * @param property name of the property to check
  */
-export function propertyNotNull<T, P extends keyof T>(property: P) {
+export function isPropertyNotNull<T, P extends keyof T>(property: P) {
```

Comment:
@romanroe you only deprecated the operator, not the predicate function here. we expose the predicate too, and some projects them. we should provide but deprecate the name too, as you did with other predicate functions ;)

Reply:
@pburgmer I dont understand. I deprecated the predicate function and didnt change the operator.

Reply:
Sorry, my fault.